Dart/README.md

```diff
@@ -0,0 +1,18 @@
+# Dart implementation
+
+## Current supportet requests
+
+- `GET *<key>*`
+- `SET *<key>* *<value>*`
+- `GETC`
+- `SETC`
```

Comment:
If your implementation implements `DELC` please specify that, if not, implement it.